pkg/collector/scheduler/job.go

```diff
@@ -77,7 +77,7 @@ func newJobQueue(interval time.Duration) *jobQueue {
 		interval:     interval,
 		stop:         make(chan bool),
 		stopped:      make(chan bool),
-		health:       health.RegisterLiveness("collector-queue"),
+		health:       health.RegisterLiveness(fmt.Sprintf("collector-queue-%vs", interval.Seconds())),
```

Comment:
Does this means we need to update the docker health probe ?

Reply:
The docker health probe ([code](https://github.com/DataDog/datadog-agent/blob/c87777041ec4c6971c4f7eb68a2908dff457ee1f/Dockerfiles/agent/probe.sh)) only runs the `agent health` command and checks its exit status, so doesn't need to be updated since it doesn't check for specific health check names.

The liveness/readiness probes in our official k8s helm chart ([code](https://github.com/DataDog/helm-charts/blob/04b6e90c04557d788dcbc34f893dbe0d027ffb9a/charts/datadog/templates/_container-agent.yaml#L199-L205)) queries the `/liveness` and `/readiness` endpoints respectively, so the same conclusion applies.